CRCSim/R/runCRCsim.R

```diff
@@ -40,21 +43,25 @@ runCRC <- function(nboot, ncores, seed = 2024){
       fb0.1 = list(direction = "both", threshold = 0.1)
     )
 
+    DT <- create.data(n, p_captures, p_strata)
+    gc(); gc()
```

Comment:
Any reason you call garbage collection twice?

Reply:
For whatever cursed reason, the first garbage collector doesn't return memory to the OS - the second one does. Unsure why, garbage collection shouldn't even need to be explicitly called, but here we are